core/src/com/po/fuck/collections/EntityCollection.java

```diff
@@ -0,0 +1,20 @@
+package com.po.fuck.collections;
+
+import com.po.fuck.Entity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class EntityCollection extends SimpleCollection<Entity> {
+    EntityCollection() {
+        super(Entity.class);
+    }
+
+    public List<Entity> getTeam (int teamTag) {
```

Comment:
Replace primary type `int` with `Class Integer`.

Reply:
For now `int` works fine